.github/workflows/unittests.yaml

```diff
@@ -57,7 +57,8 @@ jobs:
           COVERALLS_FLAG_NAME: ${{ matrix.os.name }}-${{ matrix.os.version }}-python${{ matrix.os.python }}
           COVERALLS_PARALLEL: true
         run: |
-          pip3 install --upgrade pip
+          sudo apt install python3-testresources
+          pip3 install --upgrade pip==20.0.2
```

Comment:
Which of these changes fixes the coveralls issue, and how? Could that be explained in the commit message and/or comment in this file?